tests/Integration/readme.md

```diff
@@ -50,11 +64,11 @@ docker-phpldapadmin
 2. The provided test script is executed. It also outputs results.
 3. stop.sh is executed to shut down OpenLDAP
 
-# Beware #
+autotest.sh (after ensuring certain files are present) just invokes run-test.sh with specific test files. The procedure is like above. autotest.sh will stop immediately once a test fails.
 
-This is quick solution for basically one test case. With expension this mechanism should be improved as well.
+# Beware #
 
-It does not run automatically, unless you do it. No integration with any testing framework.
+There is no integration with any testing framework.
```

Comment:
yeah, we'll need to work on that at some point... not too happy about this `AbstractIntegrationTest`